docs/quick-ingestion-guides/tableau/overview.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Overview
+---
+# Tableau Ingestion Guide: Overview
+
+## What You Will Get Out of This Guide
+
+This guide will help you set up the Tableau connector to begin ingesting metadata into DataHub.
+
+Upon completing this guide, you will have a recurring ingestion pipeline that will extract metadata from Tableau and load it into DataHub. This will include to following Tableau asset types:
+
+* Dashboards
+* Sheets 
+* Embedded DataSource
+* Published DataSource
+* Custom SQL Table
+* Embedded or External Tables
+* User
+* Workbook
+* Tag
+
+The pipeline will also extract:
+
+* **Usage statistics** to help you understand top viewed Dashboard/Chart
+* **Table- and Column-level lineage** to automatically define interdependencies between datasets and columns
```

Comment:
automatically index relationships between datasets and columns

Reply:
done